### ISSUE TITLE
The 'Program' model is sometimes referred to as 'Department' 

### ISSUE THREAD

**Issue by benaissazaki** [adilmohak/django-lms#11]
In `accounts/models.py` > `Student` model:
```
department = models.ForeignKey(Program, on_delete=models.CASCADE, null=True)
```
In the Students page :
![image](https://github.com/adilmohak/django-lms/assets/33980130/55dcb546-74a8-4e25-9fb0-ec8033746d16)

In the Add Student page :
![image](https://github.com/adilmohak/django-lms/assets/33980130/0921e387-55d5-4fe4-9079-7f32017fea71)

I do not know if it is intentional but it is confusing for users and developers.

If possible, I want to create a PR to fix it, should Program become Department or vice-versa?

**Comment by adilmohak** [comment]
Hello,

I believe that the term "Program" is more specific and indicates the area of study or major, such as "Computer Science," "Business Administration," etc. On the other hand, the term "Department" refers to the academic department or discipline within the institution, such as the "Department of Chemistry" or the "Department of History." It's a broader category that may encompass multiple programs.

In conclusion, I suggest that we use the term "Program" consistently throughout the app.

**Comment by adilmohak** [comment]
So you can contribute by changing the term "Department" to "Program" throughout the app.

Thanks :)

**Comment by benaissazaki** [comment]
Great, I'll start working on it
